web-access-control/web-access-control-test-manifest.ttl

```diff
@@ -66,55 +66,55 @@ manifest:acl-object-default
 manifest:acl-object-access-to-default
   a td:TestCase ;
   spec:requirementReference <https://solid.github.io/web-access-control-spec#access-objects> ;
-  td:reviewStatus td:unreviewed ;
+  td:reviewStatus td:approved ;
   td:preCondition "authentication", "acl" ;
   spec:testScript
     <https://github.com/solid/specification-tests/web-access-control/acl-object/container-access-to-default.feature> .
 
 manifest:protected-operation-acl-propagation
   a td:TestCase ;
   spec:requirementReference <https://solid.github.io/web-access-control-spec#authorization-evaluation-context> ;
-  td:reviewStatus td:unreviewed ;
+  td:reviewStatus td:approved ;
   td:preCondition "authentication", "acl" ;
   spec:testScript
     <https://github.com/solid/specification-tests/web-access-control/protected-operation/acl-propagation.feature> .
 
 manifest:server-wac-allow-header-exists
   a td:TestCase ;
   spec:requirementReference <https://solid.github.io/web-access-control-spec#server-wac-allow> ;
-  td:reviewStatus td:unreviewed ;
+  td:reviewStatus td:approved ;
   td:preCondition "authentication", "acl";
   spec:testScript
     <https://github.com/solid/specification-tests/web-access-control/wac-allow/header-exists.feature> .
```

Comment:
This test might be too weak to be meaningful; check header contents?

Reply:
(which is checked later; so is this test needed at all?)

Reply:
Yes, it it possibly redundant. I wondered if it was useful to separate testing the existence of the header from it's content. @kjetilk What do you think?

Reply:
I personally find such tests helpful when I'm developing, though it might be argued that it would more belong in a unit test in the software than in a conformance test suite. I'd say we keep it as a service to the community. :-)